### ISSUE TITLE
NVDA regression does not read the options to translate the page in chrome.

### ISSUE THREAD

**Issue by fernando-jose-silva** [nvaccess/nvda#6404]
To reproduce the problem.
Open a page in google chrome.
Press alt, then press shift + tabe to the button translate this page and press space.
Now return with shift + tabe to read the options.
What ever happens is that NVDA speaks only unknown.
NVDA should read the available options.
NVDA reads these options when using an older INTAL Monthly Newsletter NVDA as next 13594.
I'm currently using NVDA master 13608 and chrome 53.
The following log excerpt.

IO - inputCore.InputManager.executeGesture (15:22:39):
Input: kb (laptop): shift + tab
IO - speech.speak (15:22:39):
Speaking [u'Adicionar this p \ xe1gina the bot favorite \ xe3o ']
IO - inputCore.InputManager.executeGesture (15:22:39):
Input: kb (laptop): shift + tab
IO - speech.speak (15:22:39):
Speaking [u'Traduzir this p \ bot xe1gina \ xe3o ']
IO - inputCore.InputManager.executeGesture (15:22:40):
Input: kb (laptop): space
IO - speech._speakSpellingGen (15:22:40):
Speaking character u'espa \ xe7o '
IO - inputCore.InputManager.executeGesture (15:22:41):
Input: kb (laptop): shift + tab
IO - speech.speak (15:22:41):
Speaking [u'desconhecido ']
IO - inputCore.InputManager.executeGesture (15:22:42):
Input: kb (laptop): shift + tab
IO - speech.speak (15:22:42):
Speaking [u'desconhecido ']

**Comment by fernando-jose-silva** [comment]
The problem can be tested by reading the call that I created myself earlier in the link:
https://github.com/nvaccess/nvda/issues/6404
When reading with the arrow below NVDA reads to the line:

IO - inputCore.InputManager.executeGesture (15:22:39):
then returns to read line
To reproduce the problem.

**Comment by fernando-jose-silva** [comment]
When performing new tests I noticed that the problem occurs only on a machine with windows 7 64bit using NVDA master 13608 and chrome 53.
On a machine using NVDA master 13608 and also the chrome 53 but with the windows 10.0.14393 64bits the problem does not ever happens.

**Comment by jcsteh** [comment]
I can't reproduce this in Windows 7 or Windows 10. Are you certain Chrome is up to date in Windows 7?

**Comment by fernando-jose-silva** [comment]
I'm sure, even to ressentemente chrome uninstalled and reinstalled.
This machine with windows 7 is where I work, I have administrator rights to it.
The machine where everything works is with windows 10 is my personal machine.
I do not know if there is anything there to see, this machine with windows 7 excel when trying to edit a cell NVDA also reads only unknown, using the office in 2013.
On the machine with Windows 10 am using office 2016 and in editing cells everything is ok.

**Comment by fernando-jose-silva** [comment]
I found a way to circumvent the problem but is not the most appropriate solution.
Open chrome in the page to be translated.
Press alt, and shift + tabe to the "translate this page", press space.
Now that the subterfuge goings, move the touchpad so that the cursor is on the page NVDA will read any page of the section in question.
Now when you press chift + tabe NVDA reads dialogue and reads options as an example "translate".

**Comment by ehollig** [comment]
@fernando-jose-silva, is this still an issue? It sounds as if it is a specific issue to your particular computer. Have you tried reinstalling NVDA or tried it with the latest version of NVDA?

**Comment by fernando-jose-silva** [comment]
Problem is resolved after you format the computer. thank you

Enviado do meu iPhone

**Comment by ehollig** [comment]
Thank you for the update. Closing as worksforme
